CHANGELOG.md

```diff
@@ -1,29 +1,37 @@
-# Virtual Attributes Changelog
+# Change Log
 
-Doing our best at supporting [SemVer](http://semver.org/) with
-a nice looking [Changelog](http://keepachangelog.com).
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
 
-## Version [Unreleased] <small>tbd</small>
+## [Unreleased]
 
-## Version [3.0.0] <small>2020-09-28</small>
+## [6.0.0] - 2022-02-02
+
+* Added Rails 6.1 support
```

Comment:
```suggestion
* **BREAKING** This gem will now no longer follow Symantic Versioning, but instead follow Rails' versioning numbers in order to simplify version matches between them both.
* Added Rails 6.1 support
```

Reply:
yea. as I added Breaking, it looks better at the top

Reply:
Oh yeah..agreed.

Reply:
I think it should be in both places, so we know _when_ this versioning change was made...also commented in the statement at the top.